internal/bundleutil/bundle.go

```diff
@@ -0,0 +1,39 @@
+package bundleutil
+
+import (
+	"encoding/json"
+	"fmt"
+
+	bsemver "github.com/blang/semver/v4"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/property"
+
+	ocv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
+)
+
+func GetVersion(b declcfg.Bundle) (*bsemver.Version, error) {
```

Comment:
would it make sense for this to be part of the Bundle defn? I mean, `property` is also defined in the o-r repo. Or is this to short-circuit having to cut another release, and we'll update later?

Reply:
I don't think we have methods (or at least it wasn't my intention to have methods) on the `declcfg.*` types. Same reasoning that there are conventions in Kubernetes API types to avoid methods, which is that methods typically pull in more dependencies that clients may want to avoid.

In the future, I 100% want to make `Version` a first class field on the `BundleV2` type that we eventually add.